src/com/esotericsoftware/kryo/serializers/FieldSerializer.java

```diff
@@ -70,7 +70,7 @@
 	/** type variables declared for this type */
 	final TypeVariable[] typeParameters;
 	final Class componentType;
-	private final FieldSerializerConfig config;
+	final FieldSerializerConfig config;

```

Comment:
Semantically this should be protected probably, sorry that I haven't seen this last time.